Documentation/ceph-teardown.md

```diff
@@ -175,3 +175,16 @@ If the namespace is still stuck in Terminating state, you can check which resour
 kubectl api-resources --verbs=list --namespaced -o name \
   | xargs -n 1 kubectl get --show-kind --ignore-not-found -n rook-ceph
 ```
+
+### Remove Critical Cluster Finalizers
+
+Added Finalizers `ceph.rook.io/disaster-protection`, to protect the Critical Cluster Information from deletion during disaster scenarios and to recover from it. 
```

Comment:
```suggestion
Rook adds a finalizer `ceph.rook.io/disaster-protection` to resources critical to the Ceph cluster so that the resources will not be accidentally deleted.
```